### PR TITLE
fix publish to docker hub

### PR DESCRIPTION
**Pull request by dulmandakh**
fixed the publish action conditions, also build action will ignore version tags

### CODE REVIEW

**Comment by dulmandakh** [comment]
@gengjiawen please merge 👍

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
     paths: 
       - Dockerfile
     tags-ignore:
-      - '*.*'
```
Comment (gengjiawen) [inline .github/workflows/build.yaml:8]: Can you revert this ? *.* is enough for all cases.

**Comment by gengjiawen** [comment]
Can you go through on a full release in your fork, I am afraid we will make the git commit log ugly with too many failed tried https://github.com/dulmandakh/docker-android/actions?query=workflow%3APublish

**Comment by dulmandakh** [comment]
@gengjiawen publish action is running on https://github.com/dulmandakh/docker-android/runs/1314705293, but won't push to Docker Hub

**Comment by dulmandakh** [comment]
@gengjiawen it successfully built docker image, and here is a snippet from docker build jobs with proper tags

```sh
/usr/bin/docker buildx build --tag reactnativecommunity/react-native-android:1.0 --tag reactnativecommunity/react-native-android:latest --iidfile /tmp/docker-build-push-QYnCf2/iidfile --secret id=GIT_AUTH_TOKEN,src=/tmp/docker-build-push-QYnCf2/tmp-2709-IwHeEg27BHT1 --file ./Dockerfile https://github.com/dulmandakh/docker-android.git#tags/v1.0
13
```

**Comment by gengjiawen** [comment]
Please revert `.github/workflows/build.yaml`.

**Comment by dulmandakh** [comment]
@gengjiawen done

**Review by gengjiawen — APPROVED** [review]
LGTM

**Pull request merged**